File: C/Colleen.c
#include <stdio.h>
void ft_useless() {}
int main() {
	char *c = "#include <stdio.h>%cvoid ft_useless() {}%cint main() {%c%cchar *c = %c%s%c;%c%cprintf(c, 10, 10, 10, 9, 34, c, 34, 10, 9, 10, 9, 10, 9, 10, 10);%c%cft_useless();%c%c// useless comment inside%c}%c// useless comment outside";
	printf(c, 10, 10, 10, 9, 34, c, 34, 10, 9, 10, 9, 10, 9, 10, 10);
	ft_useless();
	// useless comment inside
}
// useless comment outside

File: PY/Colleen.py
#useless comment outside
def ft_useless():
	pass
if __name__ == "__main__":
	#useless comment inside
	ft_useless()
	code = '#useless comment outside\ndef ft_useless():\n\tpass\nif __name__ == %c__main__%c:\n\t#useless comment inside\n\tft_useless()\n\tcode = %r\n\tprint(code %% (34, 34, code, 39, 39), end=%c%c)'
	print(code % (34, 34, code, 39, 39), end='')